roles/openshift_gcp/tasks/setup_scale_group_facts.yml

```diff
@@ -1,10 +1,14 @@
 ---
+- name: Set var to exclude bootstrapped nodes
+  set_fact:
+    bootstrapped_nodes: "{{ all_nodes | default(false) | ternary([], groups['tag_ocp-bootstrap']) | default([]) }}"
```

Comment:
Let's change the 'all_nodes' to something more descriptive like 'openshift_gcp_exclude_bootstrapped_nodes' and revise the logic to the task name.